userspace/libsinsp/threadinfo.cpp

```diff
@@ -510,8 +511,7 @@ std::string sinsp_threadinfo::get_container_user() {
 
 	const auto container_id = get_container_id();
 	if(!container_id.empty()) {
-		auto table = m_inspector->m_thread_manager->get_table(
-		        sinsp_thread_manager::s_containers_table_name);
+		auto table = m_thread_manager->get_table(sinsp_thread_manager::s_containers_table_name);
```

Comment:
Umm a thread manager is a table, not sure its responsibility is to provide info about other tables... (not your code, I know)